.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "deployer/common"]
 	path = deployer/common
 	url = https://github.com/openshift/origin-integration-common
+[submodule "kibana-proxy"]
```

Comment:
I dont feel strongly about it either.  As long as it works, fine with me.  Less work.